packages/synapse-react-client/src/components/DownloadCart/DownloadCartPage.tsx

```diff
@@ -306,10 +306,7 @@ export function DownloadCartPage(props: DownloadListActionsRequiredProps) {
                   }
                   numFiles={data.numberOfFilesAvailableForDownload}
                 />
-                <AvailableForDownloadTable
-                  filesStatistics={data}
-                  refetchStatistics={refetch}
```

Comment:
Removed `refetchStatistics` and properly linked up the react-query hooks to invalidate the cache when statistics should be re-fetched